ddtrace/contrib/sanic/patch.py

```diff
@@ -0,0 +1,115 @@
+import ddtrace
+from ddtrace import config
+from ddtrace.constants import ANALYTICS_SAMPLE_RATE_KEY
+from ddtrace.ext import SpanTypes, http
+from ddtrace.http import store_request_headers, store_response_headers
+from ddtrace.propagation.http import HTTPPropagator
+from ddtrace.utils.wrappers import unwrap as _u
+from ddtrace.vendor.wrapt import wrap_function_wrapper as _w
+
+import sanic
+
+from ...internal.logger import get_logger
+
+log = get_logger(__name__)
+
+config._add("sanic", dict(service=config._get_service(default="sanic"), distributed_tracing=True))
+
+
+def _extract_tags_from_request(request):
+    tags = {}
+
+    http_method = request.method
+    if http_method:
+        tags[http.METHOD] = http_method
+
+    http_url = request.url
```

Comment:
According to https://sanic.readthedocs.io/en/latest/sanic/request_data.html the url is going to contain the query parameters which we don't want to include unless explicitly told to (as checked below). Instead this url should probably be built up from request.scheme, request.server_name, request.server_port and request.path.